sql/2019/12_Mobile_Web/12_12.sql

```diff
@@ -0,0 +1,30 @@
+#standardSQL
+
+# input types
+
+# dataset: `httparchive.pages.2019_07_01_mobile`
+# sample: `httparchive.almanac.pages_desktop_1k`
+
+CREATE TEMPORARY FUNCTION parseStructuredData(payload STRING)
+RETURNS ARRAY<STRING> LANGUAGE js AS '''
+  try {
+    var $ = JSON.parse(payload);
+    var almanac = JSON.parse($._almanac);
+    return almanac['input-elements'] && almanac['input-elements'].map(function(node) {
+        return node.type.toLowerCase();
+    });
+  } catch (e) {
+    return [];
+  }
+''';
+
+SELECT
+    flattened,
+    COUNT(flattened) AS occurence,
+    COUNT(flattened) / COUNT(url) AS occurence_perc
```

Comment:
Fixing up the percent calculation and formatting as a % value.

```suggestion
    ROUND(COUNT(flattened) * 100 / SUM(COUNT(0)) OVER (), 2) AS occurence_perc
```